app/views/locations/show.html.haml

```diff
@@ -1,23 +1,40 @@
 .level
   .level-left
-    %h1.title
-      %i.fa.fa-map-marker
-      Location
-      = @location.name
+
+    %nav.breadcrumb.has-arrow-separator.is-large
+      %ul
+        - (get_parents @location).each do |t|
```

Comment:
The brackets semantically make sense, however usually this would be written as `get_parents(@location).each ...`